gcexport.py

```diff
@@ -844,7 +914,7 @@ def main(argv):
 
         print('Fetching user stats...', end='')
         logging.info('Userstats page %s', URL_GC_USERSTATS + display_name)
-        result = http_req(URL_GC_USERSTATS + display_name)
+        result = http_req_as_string(URL_GC_USERSTATS + display_name)
```

Comment:
Again, I would suggest to use literal string interpolation (see PEP 498):
```suggestion
        result = http_req_as_string(f'{URL_GC_USERSTATS}display_name')
```
However, in this specific case it doesn't look that good ;)

Reply:
As for Python2 my suggestion should be dropped and keep the original change from the PR